rwgame/RWGame.cpp

```diff
@@ -774,15 +775,53 @@ void RWGame::renderDebugPaths(float time) {
 
         static const btVector3 color(1.f, 1.f, 0.f);
 
-        if (v->controller->targetNode && v->getCurrentVehicle()) {
-            const glm::vec3 pos1 = v->getPosition();
-            const glm::vec3 pos2 = v->controller->targetNode->position;
+        if (auto vehicle = v->getCurrentVehicle(); vehicle)
+        {
+            if (v->controller->targetNode) {
+                const glm::vec3 pos1 = v->getPosition();
+                const glm::vec3 pos2 = v->controller->targetNode->position;
 
-            btVector3 position1(pos1.x, pos1.y, pos1.z);
-            btVector3 position2(pos2.x, pos2.y, pos2.z);
+                btVector3 position1(pos1.x, pos1.y, pos1.z);
```

Comment:
Can't you use `drawLine(const glm::vec3 &from, const glm::vec3 &to, const glm::vec3 &color)` here?